feg/gateway/policydb/flow_parser.go

```diff
@@ -84,8 +84,14 @@ func getFlowDescriptionFromSplit(matches []string) (*protos.FlowDescription, err
 		Match: &protos.FlowMatch{
 			Direction: direction,
 			IpProto:   proto,
-			Ipv4Src:   src.ip,
-			Ipv4Dst:   dst.ip,
+			IpSrc:     &protos.IPAddress{
+				Version: protos.IPAddress_IPV4,
+				Address: []byte(src.ip),
+			},
+			IpDst:     &protos.IPAddress{
+				Version: protos.IPAddress_IPV4,
+				Address: []byte(dst.ip),
+			},
```

Comment:
why are these fixed to IPv4?

Reply:
I think this requires some modifications GetFlowDescriptionFromFlowString for parsing the flow string and I'd rather we do that in a separate PR. This one is primarily for the purpose of introducing the new protobuf changes and getting everything to work with them on ipv4